virt/release_notes/virt-4-18-release-notes.adoc

```diff
@@ -78,6 +78,11 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-18-monitoring"]
 === Monitoring
 
+//CNV-47048: Release note: announce the availability of new VM metrics
+* You can now view virtual machine (VM) workload metrics for allocated storage, CPU, and network resources. Admins can use these metrics to determine reource allocation and capacity planning.
++
+For a complete list of virtualization metrics, see link:https://github.com/kubevirt/monitoring/blob/main/docs/metrics.md[KubeVirt components metrics].
```

Comment:
As per this guideline, I am wondering about using or not using git link here:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites 

> You cannot link to a repository that is hosted on [www.github.com](http://www.github.com/). An exception to this rule is when autogenerated API reference content contains a link to [www.github.com](http://www.github.com/).

Reply:
We link to this list in our docs as well. The issue here is this is the only place where all of these metrics are listed. I believe other places in OCP link to this list too...